linux_os/guide/services/ssh/ssh_server/sshd_allow_only_protocol2/rule.yml

```diff
@@ -25,9 +25,11 @@ identifiers:
 references:
     nist@debian8: AC-17(7)
     nist@debian9: AC-17(7)
+    nist@debian10: AC-17(7)
     nist-csf@debian8: PR.AC-3,PR.PT-4
     anssi@debian8: NT007(R1)
     anssi@debian9: NT007(R1)
+    anssi@debian10: NT007(R1)
```

Comment:
Since all the anssi mappings are the same, please collapse into a single entry e.g. `anssi: NT007(R1)`